[jit] move some tests to test_builtins.py

suo opened the pull request:
Stack from [ghstack](https://github.com/ezyang/ghstack):
* **#29333 [jit] move some tests to test_builtins.py**

As title, trying to decrease the size of TestScript

Differential Revision: [D18393096](https://our.internmc.facebook.com/intern/diff/D18393096)

suo committed:
[jit] move some tests to test_builtins.py

As title, trying to decrease the size of TestScript

suo committed:
Update on "[jit] move some tests to test_builtins.py"


As title, trying to decrease the size of TestScript

kostmo commented:
## CircleCI build failures summary
As of commit 83577ff:
* 0/9 recognized as flaky
* 8/9 broken upstream. You may want to rebase on the [latest viable branch](https://dr.pytorch.org/master-viable-commits.html).
* 1/9 failures introduced in this PR



Here are the [reasons each build failed](https://dr.pytorch.org/commit-details.html?sha1=83577ff9a6294722225fcfe4f257ae9e55c7f9b8).

---
This comment was automatically generated by [Dr. CI](https://github.com/kostmo/circleci-failure-tracker/tree/master/docs/from-pull-request-comment).
Follow [this link to opt-out](https://dr.pytorch.org/admin/comments-opt-out.html) of these comments for your Pull Requests.


Please report bugs/suggestions on the [GitHub issue tracker](https://github.com/kostmo/circleci-failure-tracker/issues).

zdevito commented:
I don't see the same lines deleted in test_jit.py

suo committed:
Update on "[jit] move some tests to test_builtins.py"


As title, trying to decrease the size of TestScript

suo committed:
Update on "[jit] move some tests to test_builtins.py"


As title, trying to decrease the size of TestScript

suo committed:
Update on "[jit] move some tests to test_builtins.py"


As title, trying to decrease the size of TestScript

Differential Revision: [D18393096](https://our.internmc.facebook.com/intern/diff/D18393096)

suo committed:
Update on "[jit] move some tests to test_builtins.py"


As title, trying to decrease the size of TestScript

Differential Revision: [D18393096](https://our.internmc.facebook.com/intern/diff/D18393096)

suo committed:
Update on "[jit] move some tests to test_builtins.py"


As title, trying to decrease the size of TestScript

Differential Revision: [D18393096](https://our.internmc.facebook.com/intern/diff/D18393096)

suo committed:
Update on "[jit] move some tests to test_builtins.py"


As title, trying to decrease the size of TestScript

Differential Revision: [D18393096](https://our.internmc.facebook.com/intern/diff/D18393096)
